rrpproxy/utils/populate_domain_list.py

```diff
@@ -0,0 +1,24 @@
+
+def populate_domain_list(domains_in_rrpproxy_format):
+    domain_list = []
+    for key, domain in enumerate(domains_in_rrpproxy_format['property']['domain']):
+        domain_with_details = {
+            'name': domains_in_rrpproxy_format['property']['domain'][key],
+            'status': domains_in_rrpproxy_format['property']['domain status'][key],
+            'updated_date': domains_in_rrpproxy_format['property']['domain updated date'][key],
+        }
+
+        if domains_in_rrpproxy_format['property']['nameserver'][key]:
+            domain_with_details['nameserver'] = domains_in_rrpproxy_format['property']['nameserver'][key]
+        if domains_in_rrpproxy_format['property']['admincontact'][key]:
+            domain_with_details['admin_contact'] = domains_in_rrpproxy_format['property']['admincontact'][key]
+        if domains_in_rrpproxy_format['property']['techcontact'][key]:
+            domain_with_details['tech_contact'] = domains_in_rrpproxy_format['property']['techcontact'][key]
+        if domains_in_rrpproxy_format['property']['billingcontact'][key]:
+            domain_with_details['billing_contact'] = domains_in_rrpproxy_format['property']['billingcontact'][key]
+        if domains_in_rrpproxy_format['property']['ownercontact'][key]:
+            domain_with_details['owner_contact'] = domains_in_rrpproxy_format['property']['ownercontact'][key]
+
+        domain_list.append(domain_with_details)
+
+    return domain_list
```

Comment:
You change the property names in this function, not sure if that is the nicest way to go. It's probably quite confusing for people. I like your naming a lot better for sure, however I'd still advocate for consistency.

| Incoming property   | Outgoing key    |
| ------------------- | --------------- |
| domain              | name            |
| domain status       | status          |
| domain updated date | updated_date    |
| nameserver          | nameserver      |
| admincontact        | admin_contact   |
| techcontact         | tech_contact    |
| billingcontact      | billing_contact |
| ownercontact        | owner_contact    |